src/main/java/io/github/binaryoverload/JSONConfig.java

```diff
@@ -52,6 +52,8 @@
  */
 public class JSONConfig {
 
+    private int mode; // 0 = File, 1 = String, 2 = Input Stream, -1 = JsonObject
```

Comment:
why even have this? You could just set a Reader for all of these.

Reply:
The mode determines the read i use.

Reply:
Yeah why bother, you can use a Reader for all of these. No need for a weird mode thing and an Object file